### PR TITLE
docs: update screenshots in appstore to match Nc30 styles

### LINKED ISSUE #13008: Update screenshots for the appstore
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

Ref https://apps.nextcloud.com/apps/spreed

The screenshots in https://github.com/nextcloud/spreed/blob/main/appinfo/info.xml#L51-L55 should be updated for the final

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fix #13008

### 🖼️ Screenshots / Screencasts

image

image
image

### 🏁 Checklist

- [x] 🌏 Tested with different browsers / clients:
  - [ ] Chromium (Chrome / Edge / Opera / Brave)
  - [ ] Firefox
  - [ ] Safari
  - [ ] Talk Desktop
  - [ ] Not risky to browser differences / client
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: docs/chat.png
Comment (ShGKme) [inline docs/chat.png:1]: image

Comment (Antreesy) [reply docs/chat.png:1]: Fixed

image

File: docs/call-in-action.png
Comment (DorraJaouad) [inline docs/call-in-action.png:1]: Looks like a duplicate and not used elsewhere

File: docs/file-darg-and-drop.png
Comment (DorraJaouad) [inline docs/file-darg-and-drop.png:1]: file-**drag**....

**Review by ShGKme — APPROVED** [review]

File: docs/video-verfication.png
Comment (DorraJaouad) [inline docs/video-verfication.png:1]: video-**verification**...

Comment (DorraJaouad) [reply docs/video-verfication.png:1]: and maybe it's not needed to show clipboard toast message?

File: docs/call-in-action.png
Comment (Antreesy) [reply docs/call-in-action.png:1]: Maybe there is a place we are unaware of (where `raw.githubusercontent.com/...` URL is used)
That's why I don't want to remove duplicate .png and rename files

File: docs/video-verfication.png
Comment (Antreesy) [reply docs/video-verfication.png:1]: Why not?
It's a minor feature representation (same as typing indicator, markdown, call reactions)

File: docs/call-in-action.png
Comment (DorraJaouad) [reply docs/call-in-action.png:1]: But the file name would be the same even if it is used in URL

Comment (Antreesy) [reply docs/call-in-action.png:1]: Extension is different, so as URL
Do you know why png file was added in the first place then? And why it wasn't removed yet? I don't, so I don't want to take risk for deleting 😁

Comment (DorraJaouad) [reply docs/call-in-action.png:1]: Safe play

**Review by DorraJaouad — APPROVED** [review]

**Pull request merged**